src/credentials/GroupDataProviderImpl.h

```diff
@@ -215,8 +212,8 @@ class GroupDataProviderImpl : public GroupDataProvider
     };
     CHIP_ERROR RemoveEndpoints(FabricIndex fabric_index, GroupId group_id);
 
-    chip::PersistentStorageDelegate & mStorage;
-    bool mInitialized = false;
+    chip::PersistentStorageDelegate * mStorage = nullptr;
+    bool mInitialized                          = false;
```

Comment:
Can be replaced by the fact that `bool IsInitialized() { return (mStorage == nullptr); }`

Reply:
Fixed as suggested.